### ISSUE TITLE
window doesn't reflect accent color from windows accessibilities settings

### ISSUE THREAD

**Issue by PiDiBi** [microsoft/vscode#141969]
Issue Type: <b>Bug</b>

window title doesn't have accent color when selected 

VS Code version: Code 1.63.1 (fe719cd3e5825bf14e14182fddeb88ee8daf044f, 2021-12-14T02:13:54.292Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) W-2133 CPU @ 3.60GHz (12 x 3600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.59GB (51.65GB free)|
|Process Argv|. --crash-reporter-id 911ca48b-1699-4ea6-b305-0d6360bee331|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-azurefunctions|ms-|1.6.0
vscode-azureresourcegroups|ms-|0.4.0
vscode-docker|ms-|1.19.0
csharp|ms-|1.24.0
remote-containers|ms-|0.209.6
remote-wsl|ms-|0.63.13
azure-account|ms-|0.9.11
powershell|ms-|2021.12.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392cf:30425750
pythontb:30283811
pythonvspyt551:30345470
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30429731
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30428979
pythonvs932:30410667
vscop341:30404997
vsrem710:30416614

```

</details>

**Comment by isidorn** [comment]
This is currently not supported. Though it is a fair feature request. Assigning to backlog for now since we currently have no plans to work on this.

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!

**Comment by RokeJulianLockhart** [comment]
> https://github.com/microsoft/vscode/issues/141969#issuecomment-1297265130

@VSCodeTriageBot, this is an annoyance for me too, using KDE Plasma 6. Accent colour customization is solely becoming more prevalent as time progresses.
